external-crates/move/crates/move-compiler/src/typing/ast.rs

```diff
@@ -191,7 +191,7 @@ pub enum UnannotatedExp_ {
     Builtin(Box<BuiltinFunction>, Box<Exp>),
     Vector(Loc, usize, Box<Type>, Box<Exp>),
 
-    IfElse(Box<Exp>, Box<Exp>, Box<Exp>),
+    IfElse(Box<Exp>, Box<Exp>, Option<Box<Exp>>),
```

Comment:
We don't need this in typing just for the error message, but I am persisting it for lints